_includes/master/reqs-sys.md

```diff
@@ -31,19 +31,18 @@ to connect to, we provide instructions in the [installation documentation](./ins
 
 ## Network requirements
 
-{{site.prodname}} requires the network to allow the following types of traffic.
-
-| Traffic | Protocol | Port |
-| ------- | -------- | ---- |
-| BGP     | TCP      | 179  |
-| IPIP\*  | 4        | n/a  |
-
-\* {% if include.orch == "OpenStack" %} If your compute hosts connect directly
-   and don't use IPIP, you don't need to allow IPIP traffic. {% else %} Our
-   manifests enable IPIP by default.  If you disable IPIP, you won't need to
-   allow IPIP traffic. {% endif %} Refer to [Configuring
-   IP-in-IP](../../usage/configuration/ip-in-ip) for more information.
-
-> **Tip**: On GCE, you can allow this traffic using firewall rules. In AWS, use
-> EC2 security group rules.
-{: .alert .alert-success}
+Ensure that your hosts and firewalls allow the following traffic.
+
+| Configuration | Host | Connection type | Port/protocol |
+|---------------|------|-----------------|---------------|
+{% if include.orch == "Kubernetes" or include.orch == "OpenShift" %}| All | kube-apiserver host | Incoming | The `targetPort` returned by `kubectl get svc kubernetes -o yaml` |
```

Comment:
I think its important this page is easily consumable when viewed raw, so that its easier to process for maintainers. This is really quite difficult to read with the jekyll includes as written currently. Two suggestions:
- Split this over multiple lines by using [liquid whitespace control](https://shopify.github.io/liquid/basics/whitespace/). We use this in the manifest inclusions already, see [here](https://github.com/projectcalico/calico/blob/master/_includes/master/manifests/calico-node.yaml#L58). 
- Don't template within an individual row.

Example: 
```
{% if include.orch == "Kubernetes" or include.orch == "OpenShift" -%}
| All | kube-apiserver host | Incoming | The `targetPort` returned by `kubectl get svc kubernetes -o yaml` |
{% endif -%}

{% if include.orch == "Kubernetes" %}
| etcd datastore | etcd hosts | Incoming | Varies according to your etcd configuration |
{%- else %}
| All | etcd hosts | Incoming | Varies according to your etcd configuration |
{%- endif %}
```

(my dashes may not be 100% correct but should be close)

Reply:
OK acknowledged...sorry about that. I will fix it. @ozdanborne @neiljerram @caseydavenport